lib/assessment.rb

```diff
@@ -43,6 +48,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
+    memo[:lines] = memo[:lines.include] + 1 ("\n")
```

Comment:
Sorry, this one's not right. Here are two possible answers.
`File.open(filename).reduce('') { |a, e| a << e.chomp + ' ' }.strip`
OR
```ruby
File.open(filename).each_with_object('') do |line, memo|
  memo << line.chomp + ' '
end.strip
```